plugins/source/aws/client/client.go

```diff
@@ -33,8 +33,8 @@ type Client struct {
 	Backend              state.Client
 	specificRegions      bool
 	Spec                 *Spec
-	// Do not rely on this field, it will be removed once https://github.com/aws/aws-sdk-go-v2/issues/2163 is resolved
```

Comment:
Is this commend not longer true? Will we keep AWSConfig?

Reply:
We will need to because the `initService` function uses it to initialize the service:
```
func (s *Services) InitService(awsConfig *aws.Config, service string) {
	c := awsConfig.Copy()
```